metadata.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "voxpupuli-posix_acl",
+  "name": "puppet-posix_acl",
```

Comment:
This is the important bit.  The other changes are mostly for consistency with other modules.

Reply:
LGTM